### PR TITLE
Release 1.8-rc1

### PR DESCRIPTION
**Pull request by joeholley**
### Release Notes
**Feature Highlights** 
- Move to latest Bitnami Redis helm chart [pr/1581](https://github.com/googleforgames/open-match/pull/1581) https://github.com/googleforgames/open-match/pull/1541 (ashutosji)
- Remove podsecuritypolicy from the Open Match helm chart Kubernetes manifests. This enables the helm chart to be installed on Kubernetes versions >= 1.25 https://github.com/googleforgames/open-match/pull/1572 https://github.com/googleforgames/open-match/pull/1540 (mridulji)
- Fix spelling in one of the examples https://github.com/googleforgames/open-match/pull/1585 (denizsurmeli)
- Unlock redsync locks for backfill functions https://github.com/googleforgames/open-match/pull/1578 (glindstedt)

**Breaking Changes** 
- Helm chart no longer supports installing Jaeger. The golang library this support was dependant on has been deprecated and moving to the latest version of other dependancies broke it. Any future tracing support will need to implement the OpenTelemetry standard (not included in this release)

**Security Fixes**
- bump golang modules flagged for vulnerabilities by dependabot https://github.com/googleforgames/open-match/pull/1569 (mridulji)

**Enhancements**
- Bump golang to 1.21
- Bump protoc version to 24.0
- Bump gRPC Gateway to 2.16.2
- Bump Kubectl version to 1.27.3
- Bump Helm version to 3.12.3
- Move CI/CD process to kaniko build cache to speed it up a bit

**Usage Requirements**
Tested against Kubernetes Version 1.24 and 1.25
Helm chart tested using Helm Version 3.12.3
Modifying Open Match core requires Golang Version >= v1.21

Will likely still need some changes to true up the examples, but I won't get around to testing them this week.

### CODE REVIEW

**Comment by markmandel** [comment]
Just checking if you want review yet or not, since conflicts + breaking build?

**Comment by joeholley** [comment]
Ready for review. I can't troubleshoot why the trigger check never passes (I don't have admin perms); AFAIK all it does is trigger the build, which /does/ complete and pass. No idea why the trigger never goes green.

File: testing/e2e/backfill_test.go [hunk 1]
```diff
@@ -559,61 +558,87 @@ func TestBackfillGenerationMismatch(t *testing.T) {
...
-	resp, err = stream.Recv()
-	require.Nil(t, resp)
-	require.Error(t, err)
-	require.Equal(t, io.EOF.Error(), err.Error())
-
-	_, err = om.Frontend().GetBackfill(ctx, &pb.GetBackfillRequest{BackfillId: b1.Id})
-	require.Error(t, err)
-	require.Equal(t, fmt.Sprintf("rpc error: code = NotFound desc = Backfill id: %s not found", b1.Id), err.Error())
-}
+// joeholley: commented out 2023.08.08: this test is flakey and is frequently failing the build with the following error:
```
Comment (markmandel) [inline testing/e2e/backfill_test.go:561]: My gut says, delete this in the PR, and file a GitHub issue with the contents of your comments and the original tests (or just point to them in previous commits).

File: testing/e2e/cluster_test.go [hunk 2]
```diff
@@ -90,18 +86,25 @@ func TestMain(m *testing.M) {
...
-	cfgMemory := viper.New()
-	cfgMemory.SetConfigType("yaml")
-	err = cfgMemory.ReadConfig(strings.NewReader(configFile))
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	require.Equal(t, cfgMemory.AllSettings(), cfg.AllSettings())
-}
+// joeholley: commented out 2023.08.15: this test can no longer work as-is; the standalone
```
Comment (markmandel) [inline testing/e2e/cluster_test.go:89]: If this test doesn't work, then let's just delete it.

**Review by markmandel — COMMENTED** [review]
Exciting!

File: testing/e2e/backfill_test.go (hunk 1 above)
Comment (joeholley) [reply testing/e2e/backfill_test.go:561]: SGTM

File: testing/e2e/cluster_test.go (hunk 2 above)
Comment (joeholley) [reply testing/e2e/cluster_test.go:89]: SGTM

**Comment by Kaszanas** [comment]
If you need any assistance, I have experience in running the OpenMatch examples. I wasn't able to see any activity in logs after following the custom evaluator instructions. This is mostly related #1580

Additionally, some of the tutorial steps provide code for running multiple steps at the same time in the l command line or use newline indication which comes with it's own drawbacks dependent on the shell that it's ran in.

File: cloudbuild.yaml [hunk 3]
```diff
@@ -48,23 +45,31 @@
 
 steps:
 - id: 'Docker Image: open-match-build'
-  name: gcr.io/cloud-builders/docker
-  args: ['build', '-t', 'gcr.io/$PROJECT_ID/open-match-build', '-f', 'Dockerfile.ci', '.']
+  name: gcr.io/kaniko-project/executor:latest
```
Comment (markmandel) [inline cloudbuild.yaml:48]: Oh nice use of kaniko. I might steal this.

**Review by markmandel — APPROVED** [review]
Approved.

I had a quick look for the issue to be created for the e2e test you had to delete, and couldn't see it, but I may not be looking too hard as I run off to NEXT, so in case it's not there, please add it 😃

But otherwise, good to go!

**Comment by joeholley** [comment]
> I had a quick look for the issue to be created for the e2e test you had to delete, and couldn't see it, but I may not be looking too hard as I run off to NEXT, so in case it's not there, please add it 😃

(For tracking)
https://github.com/googleforgames/open-match/issues/1592
https://github.com/googleforgames/open-match/issues/1591

**Pull request merged**